### ISSUE TITLE
LibIconPath is not being packaged properly, causing Lua errors ingame

### ISSUE THREAD

**Issue by eTzmNcbkrng** [SacredDuckwhale/TotalAP#44]
I just updated to latest version from Curse (Twitch), .toc show this `## Version: 1.5.3 (r27-alpha)`.
I am getting an error every time I mouse over the bars, mousing over the spec buttons does not error, but also does not display current level or progress to next level.

Disabled all other addons, error persists;

```140x TotalAP\Core\GUI\Tooltips.lua:92: attempt to call global 'LibIconPath_getPath' (a nil value)
TotalAP\Core\GUI\Tooltips.lua:92: in function <TotalAP\Core\GUI\Tooltips.lua:37>
TotalAP\Core\GUI\Tooltips.lua:142: in function <TotalAP\Core\GUI\Tooltips.lua:140>

Locals:
self = TotalAP_DefaultView_ProgressBar3 {
 0 = <userdata>
 texture = <unnamed> {
 }
}
button = true
hide = nil
specID = 3
fqcn = "Duglazerp - Saurfang"
_ = "Warrior"
_ = "WARRIOR"
classID = 1
itemID = 128289
artifactName = "Scale of the Earth-Warder"
numTraitsPurchased = 51
maxAvailableAP = 245185964
tier = 2
maxKnowledgeLevel = 55
maxAttainableRank = 51
progressPercent = 47.151146923077
knowledgeLevel = 0
maxAttainableKnowledgeLevel = 0
shipmentsReady = 0
shipmentsTotal = 0
timeLeft = nil
timeLeftString = nil
settings = <table> {
 enabled = true
 scanBank = true
 actionButton = <table> {
 }
 numberFormat = "enUS"
 verbose = true
 specIcons = <table> {
 }
 autoHide = true
 infoFrame = <table> {
 }
 showLoginMessage = true
 tooltip = <table> {
 }
 debugMode = false
}
relicRecommendations = <table> {
 1 = 238077
 2 = 188635
 3 = 203225
}
(for generator) = <function> defined =[C]:-1
(for state) = <table> {
 1 = 238077
 2 = 188635
 3 = 203225
}
(for control) = 1
priority = 1
spellID = 238077
name = "Bastion of the Aspects"
_ = ""
icon = 236307
(*temporary) = nil
(*temporary) = 236307
(*temporary) = "attempt to call global 'LibIconPath_getPath' (a nil value)"
TotalAP = <table> {
 Utils = <table> {
 }
 Cache = <table> {
 }
 DB = <table> {
 }
 Controllers = <table> {
 }
 Scanner = <table> {
 }
 ArtifactInterface = <table> {
 }
 Settings = <table> {
 }
 L = <table> {
 }
 Debug = <function> defined @TotalAP\Core\Init.lua:77
 DBHandler = <table> {
 }
 Addon = <table> {
 }
 GetLocaleNumberFormat = <function> defined @TotalAP\Locales\NumberFormats.lua:454
 EventHandlers = <table> {
 }
 versionString = "1.5.3 (r27-alpha)"
 GUI = <table> {
 }
 eventStates = <table> {
 }
 artifactCache = <table> {
 }
 ChatMsg = <function> defined @TotalAP\Core\Init.lua:88
 inventoryCache = <table> {
 }
 bankCache = <table> {
 }
}
L = <table> {
 This spec is currently active = "This spec is currently active"
 TotalAP - Artifact Power Tracker = "TotalAP - Artifact Power Tracker"
 Number format set to %s = "Number format set to %s"
 Artifact Knowledge Level: %d = "Artifact Knowledge Level: %d"
 Load default settings (will overwrite any changes made) = "Load default settings (will overwrite any changes made)"
 %s Artifact Power in bank = "%s Artifact Power in bank"
 Type %s to reset all currently ignored specs for this character = "Type %s to reset all currently ignored specs for this character"
 Debug mode disabled. = "Debug mode disabled."
 You cannot use keybinds to change the display while in combat. = "You cannot use keybinds to change the display while in combat."
 Icons are now hidden. = "Icons are now hidden."
 Items stored in the bank are now being ignored = "Items stored in the bank are now being ignored"
 Button glow effect enabled. = "Button glow effect enabled."
 Maximum number of traits unlocked = "Maximum number of traits unlocked"
 [List of available commands] = "[List of available commands]"
 %.2f%% towards Rank %d = "%.2f%% towards Rank %d"
 Toggle spell overlay notification (glow effect) when new traits are available = "Toggle spell overlay notification (glow effect) when new traits are available"
 Progress report enabled. = "Progress report enabled."
 Toggle visibility while items are unable to be used = "Toggle visibility while items are unable to be used"
 Relic recommendations enabled. = "Relic recommendations enabled."
 Action button text enabled. = "Action button text enabled."
 Display will now remain visible at all times. = "Display will now remain visible at all times."
 Next in: %s = "Next in: %s"
 Action button is now hidden. = "Action button is now hidden."
 Login message is now hidden. = "Login message is now hidden."
 Resets ignored specs for the currently active character = "Resets ignored specs for the currently active character"
 All displays are now being shown. = "All displays are now being shown."
 Toggle icon and text display for artifact power progress = "Toggle icon and text display for artifact power progress"
 Item counter disabled. = "Item counter disabled."
 D
```

**Comment by rdw-software** [comment]
Yep, can confirm that the CF packager has an issue with a library I added (see [here](https://wow.curseforge.com/projects/libiconpath/issues/4)). I notified the author, but I will be removing the library anyway because it is literally doubling the addon's size and most of it isn't even needed.

**Comment by rdw-software** [comment]
Should be fixed with 533888dbe38ba58153480fc2e67619689ac6059c (newest alpha).
